src/main/resources/reference.conf

```diff
@@ -0,0 +1,4 @@
+
+mesos-actor {
+  agent-stats-ttl = 1 minutes //time that agent stats remain valid without an update; incrase time in case offer cycle is slow; decrease time ignore offer data that may be stale
```

Comment:
should we document that the TTL should be a multiple of `10s` ?

Reply:
I'm basically asking if there will be issues with someone configures this with `65 seconds`

Reply:
There's no guarantee that the ttl will be precise - even if you set it for a multiple of 10s, it may be pruned up to 10s later that the expiration time based on the ttl. If this precision is important we could make the pruning period configurable, or just make it lower?

Reply:
I made the period configurable with default of `5 seconds`